### PR TITLE
homework_5

### PR DESCRIPTION
**Pull request by artsiomshevel**

### CODE REVIEW

File: Homework_5/homework_5.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+//   Результат должен быть округлен до 1 знака после запятой методом .toFixed(), про который надо почитать самим ;)
+
+let bytesValue = 1_454_548
+// let kbInbytes = (bytesValue/1024).toFixed(1)
+// let mbInbytes = (bytesValue/1024/1024).toFixed(1)
+// let gbInbytes = (bytesValue/1024/1024/1024).toFixed(1)
+// let tbInbytes = (bytesValue/1024/1024/1024/1024).toFixed(1)
+
+if (bytesValue < 1024) {
```
Comment (VladimirLQA) [inline Homework_5/homework_5.js]: Use `while` loop for this task

File: Homework_5/homework_5.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        row += ' '
+    }
+    //loop for *
+    for (let k = 0; k < (2*i + 1); k++) {
+        row += '*'
+    }
+    console.log(row)
+}
+
+//ромб ???
```
Comment (VladimirLQA) [inline Homework_5/homework_5.js]: ??? :)

File: Homework_5/homework_5.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+
+// 3.  Вам нужно вывести в консоль числа от 1 до 100.
+//     Если число делится без остатка на 3, то выведете в консоль “число - делится на 3”.
+//     Если число делится на 5 без остатка, то то выведете в консоль “число - делится на 5”.
+//     Если число делится и на 3 и на 5 без остатка, то то выведете в консоль “число - делится и на 3 на 5”.
+//     Число 15 делится без остатка на 3 и на 5 -- пример сообщения в консоле.
+for (let i = 1; i <= 100; i++) {
+    if (i % 3 === 0) {
```
Comment (VladimirLQA) [inline Homework_5/homework_5.js]: Try to change order of `if...else` blocks
Following task description for number `15` we should get such log 
`Число 15 делится без остатка на 3 и на 5`, with your implementation I've got `число 15 делится без остатка на 3`

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

**Comment by VladimirLQA** [comment]
Please, add the solution for 3-rd task or maybe you've just forgot to add it to commit ...

File: Homework_5/homework_5.js (hunk 2 above)
Comment (artsiomshevel) [reply Homework_5/homework_5.js]: @VladimirLQA  task 2.2 is making me depressed :)

File: Homework_5/homework_5.js (hunk 1 above)
Comment (artsiomshevel) [reply Homework_5/homework_5.js]: fixed @VladimirLQA

File: Homework_5/homework_5.js (hunk 3 above)
Comment (artsiomshevel) [reply Homework_5/homework_5.js]: fixed @VladimirLQA

File: Homework_5/homework_5.js (hunk 2 above)
Comment (artsiomshevel) [reply Homework_5/homework_5.js]: fixed @VladimirLQA

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**